src/app/device-search/device-search.component.html

```diff
@@ -1,39 +1,54 @@
 <div id="main">
   <div class="container-fluid content">
-    <div class="row justify-content-center">
-      <div class="col-lg-12">
-        <h2 class="text-center">Devices</h2>
+    <div class="row justify-content-center header">
+      <div class="col-lg-7">
+        <h2>Devices</h2>
+        <p>Are you looking for a specific device and/or someone who is familiar with it? Explore our latest devices!</p>
```

Comment:
Use of word device is too often.
Rephrase: ... Explore our latest entries!

Reply:
Good point! I changed it.